### PR TITLE
Fix commit install failure

### PR DESCRIPTION
**Pull request by psergee**
This is fast head-on solution, no code duplication removed.

### CODE REVIEW

**Review by oleg-jukovec — APPROVED** [review]
Thank you for the patch! Does it works https://stackoverflow.com/questions/15124430/how-to-checkout-old-git-commit-including-all-submodules-recursively/44249558#44249558 ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```
Comment (LeonidVas) [inline CHANGELOG.md:8]: Changes not related with the commit title. Please move them to a separate commit.

**Review by LeonidVas — APPROVED** [review]
Hi! Thank you for the patchset.

Comment body not related with the comment title.
```
@TarantoolBot document
Title: abitilty to install tarantool/tt using commit hash and pull request number.

A user can install tarantool and tt using hash or pull request id.
For example:
```
tt install tarantool 00a9e59
tt install tt pr/582
```
```

**Comment by psergee** [comment]
> Thank you for the patch! Does it works https://stackoverflow.com/questions/15124430/how-to-checkout-old-git-commit-including-all-submodules-recursively/44249558#44249558 ?

It works perfectly, thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (psergee) [reply CHANGELOG.md:8]: Moved.

**Comment by psergee** [comment]
> Hi! Thank you for the patchset.
> 
> Comment body not related with the comment title.
> 
> ```
> @TarantoolBot document
> Title: abitilty to install tarantool/tt using commit hash and pull request number.
> 
> A user can install tarantool and tt using hash or pull request id.
> For example:
> 
> tt install tarantool 00a9e59
> tt install tt pr/582
> ```

Removed.

**Review by oleg-jukovec — APPROVED** [review]
